***E-Invoice Data for Wholesale marketplace***

This data has the following columns and rows

**Columns** <br>
Invoice_ID - Denotes the invoice Id for the transaction <br>
Seller_Name - Company who sells the services/product <br>
Buyer_Name - Company that buys the services/product <br>
Description - Describes the type of services or product <br>
Quantity - denoted the number of items sold <br>
Unit_price - price per item <br>
Terms_of_payment - Duration in the buyer is expected to pay for the services rendered by the seller <br>
Total_amount - Total amount of the transaction <br>

**Rows**
Unique Transactions which will have all the information denoted above

In [5]:
import pandas as pd

In [7]:
df = pd.read_csv(r'C:\Users\User\Downloads\E-Invoice_Mock_Data.csv')
df.head(10)

,Invoice_ID,Seller_Name,Buyer_Name,Description,Quantity,Unit_Price,Terms_of_Payment,Total_Amount
0,INV-00001,Pro Services,Cozy Comforts,Home Decor,449,1.0,Net 15,449.0
1,INV-00002,Urban Essentials,Dynamic Buyers,Biodegradable Packaging,427,300.0,Net 15,128100.0
2,INV-00003,HomeStyle Co.,WorldWide Supplies,Groceries,233,2000.0,Immediate,466000.0
3,INV-00004,Smart Devices Ltd.,Premium Services,Sports Equipment,245,80.0,Net 15,19600.0
4,INV-00005,Eco Supplies Co.,Cozy Comforts,Imported Goods,258,500.0,Net 30,129000.0
5,INV-00006,Dynamic Supplies,BetterLife Stores,Smart Home Devices,434,900.0,Net 60,390600.0
6,INV-00007,Office World,BetterLife Stores,Groceries,32,2000.0,Net 60,64000.0
7,INV-00008,TopTech Gadgets,Metro Systems,Daily Essentials,498,300.0,Net 15,149400.0
8,INV-00009,Global Imports,FastTrack Corp.,Office Chairs,269,200.0,Net 45,53800.0
9,INV-00010,Power Tools Co.,Premium Services,Art Supplies,265,50.0,Net 60,13250.0


1. Total Sales Insights
Highest Sale Amount:
The highest sale is from HomeStyle Co. to WorldWide Supplies for groceries, totaling 466,000.
Lowest Sale Amount:
The lowest sale is from Pro Services to Cozy Comforts for home decor, totaling 449.